__tests__/libs/ontouml2db/mapper.test.ts

```diff
@@ -0,0 +1,1595 @@
+          Cardinality.C1,
+          'supply_contract',
+          Cardinality.C0_N,
+        ),
+      )
+      .addRelationship(
+        new RelationshipChecker(
+          'organization',
+          Cardinality.C0_1,
+          'supply_contract',
+          Cardinality.C0_N,
+        ),
+      )
+      .addRelationship(
+        new RelationshipChecker(
+          'organization',
+          Cardinality.C1,
+          'enrollment',
+          Cardinality.C0_N,
+        ),
+      )
+
+      .addTracker(new TrackerChecker('NamedEntity', 'person'))
+      .addTracker(new TrackerChecker('NamedEntity', 'organization'))
+      .addTracker(new TrackerChecker('Person', 'person'))
+      .addTracker(new TrackerChecker('Organization', 'organization'))
+      .addTracker(new TrackerChecker('BrazilianCitizen', 'person'))
+      .addTracker(new TrackerChecker('ItalianCitizen', 'person'))
+      .addTracker(new TrackerChecker('Child', 'person'))
+      .addTracker(new TrackerChecker('Adult', 'person'))
+      .addTracker(new TrackerChecker('Employee', 'person'))
+      .addTracker(new TrackerChecker('Customer', 'person'))
+      .addTracker(new TrackerChecker('Customer', 'organization'))
+      .addTracker(new TrackerChecker('PersonalCustomer', 'person'))
+      .addTracker(new TrackerChecker('CorporateCustomer', 'organization'))
+      .addTracker(new TrackerChecker('Employment', 'employment'))
+      .addTracker(new TrackerChecker('SupplyContract', 'supply_contract'))
+      .addTracker(new TrackerChecker('Contractor', 'organization'))
+      .addTracker(new TrackerChecker('PrimarySchool', 'organization'))
+      .addTracker(new TrackerChecker('Hospital', 'organization'))
+      .addTracker(new TrackerChecker('Enrollment', 'enrollment'));
+
+    let result = gChecker.check();
+
+    if (result != '') {
+      console.log(mapper.getSourceGraph().toString());
+      console.log(mapper.getTargetGraph().toString());
+      expect(result).toBe('');
+    }
+  });
+
+  test('001 Simple Flattening', () => {
+    let mapper = new Transformation2DB(test_001);
+    mapper.setStandardizeDatabaseNomenclature(true);
+    mapper.doMapping();
+
+    let gChecker = new GraphChecker()
+      .setTransformation(mapper)
+      .addNode(
+        new NodeChecker('person')
+          .addProperty(new PropertyChecker('person_id', false))
+          .addProperty(new PropertyChecker('name', false))
+          .addProperty(new PropertyChecker('birth_date', false)),
+      )
+      .addTracker(new TrackerChecker('NamedEntity', 'person'))
+      .addTracker(new TrackerChecker('Person', 'person'));
+
+    let result = gChecker.check();
+
+    if (result != '') {
+      console.log(
+        '************ SOURCE GRAPH: \n' + mapper.getSourceGraph().toString(),
+      );
+      console.log(
+        '************ TARGET GRAPH: \n' + mapper.getTargetGraph().toString(),
+      );
+      expect(result).toBe('');
+    }
+  });
+
+  test('002 Flatting with Duplicate Attribures', () => {
```

Comment:
The number is the index for referencing the software artifact (use case, test case, ...) wherever it is used, without changing. For example: the "Register Customer" use case may change in the design or development phase so that the most appropriate name becomes "Manage Customer". This change generates maintenance wherever the description is used, which would not happen if the name of the use case was "UC001". The same reasoning applies to test documents or any other system artifact.

However, there is no way to disagree with the readability of the nomenclature. So, as agreed with Claudenir, I will add to the number the brief description of the test case, eg "001_simple_felttening", and internally, I will add more details.